tinychat/vlm_demo_new.py

```diff
@@ -197,20 +211,20 @@ def main(args):
         "--model_type", type=str, default="LLaMa", help="type of the model"
     )
     parser.add_argument(
-        "--model-path", type=str, default="/data/llm/checkpoints/llava/llava-v1.5-7b"
+        "--model-path", type=str, default="/home/yuming/workspace/llm-awq_1/tinychat/VILA1.5-13b-AWQ"
     )
     parser.add_argument(
         "--quant-path",
         type=str,
-        default="/data/llm/checkpoints/llava/llava-v1.5-7b-w4-g128-awq.pt",
+        default="/home/yuming/workspace/llm-awq_1/tinychat/VILA1.5-13b-AWQ/llm/vila-1.5-13b-w4-g128-awq-v2.pt",
     )
     parser.add_argument(
         "--precision", type=str, default="W4A16", help="compute precision"
     )
     parser.add_argument(
         "--image-file",
         type=str,
-        default="https://llava.hliu.cc/file=/nobackup/haotian/code/LLaVA/llava/serve/examples/extreme_ironing.jpg",
+        default="/home/yuming/workspace/VILA/demo_images/av.png",
```

Comment:
Please be careful about the default paths. Ideally the user name should not appear in the paths.